lib/Doctrine/DBAL/Portability/Connection.php

```diff
@@ -13,6 +13,20 @@
  */
 class Connection extends \Doctrine\DBAL\Connection
 {
+    /**
+     * Force column names to upper case.
+     *
+     * @see \PDO::CASE_UPPER
+     */
+    const CASE_UPPER = 1;
```

Comment:
Same like fetch mode - I'd prefer i.e. ColumnCasing enum-like structure.

Reply:
I'm fine with enum-like classes and non-abbreviated names. Not sure if we want bit masks and we do need to introduce them in `2.x` to ease the migration.